### DISCUSSION TITLE
Opening a shell in Interlisp which runs in a container

### DISCUSSION THREAD

**Discussion by reflektoin** [Interlisp/medley#688] [Q&A] [answered]
**EDIT: This isn't really an issue for me at the moment. I have started to use the online version**

Hi, Thank you for this project! By following the [Docker guide here](https://github.com/Interlisp/medley/wiki/Running-with-Docker), I was able to get Interlisp running easily.

But I got stuck when I tried to open a terminal in Interlisp with command (IL:CHAT 'SHELL). I got a message 
`[Connection closed by remote host]`

My current setup is that I'm running Interlisp Docker container on a Ubuntu server. From my own computer I create ssh-tunnel to the server and use either SSVNC or xtightvnc to connect to Interlisp.

Does someone have tips how to proceed debugging this?

And thanks again for making this!

**Comment by masinter** [comment, marked as answer]
Apparently I made this Discussion into an issue #765 since we're not tracking Discussions. Short answer is that the Docker instructions need work.
